src/main/java/com/structurizr/export/plantuml/StructurizrPlantUMLExporter.java

```diff
@@ -351,7 +351,7 @@ protected void writeElement(ModelView view, Element element, IndentingWriter wri
                 shape = plantUMLShapeOf(view, elementInstance.getElement());
 
                 if (StringUtils.isNullOrEmpty(url)) {
-                    url = element.getUrl();
+                    url = elementInstance.getUrl();
```

Comment:
I wonder if this was supposed to be the instance Url - just noticed this when going through the code.